Week1/hw3/trip.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="trip.css" />
+    <title>두근두근 여행</title>
+</head>
+<body>
+    <section>
+        <header>
+            <h1 class="title">💜 두근두근 유럽 여행</h1>
+        </header>
+        <div class="preview">
+            <ul>
+                <li >
+                    <img src="./images/london3.jpg" alt="아이스크림"/>
+                </li>
+                <li>
+                    <img src="./images/london2.jpg" alt="소호거리"/>
+                </li>
+                <li>
+                    <img src="./images/paris1.jpg" alt="아르떼뮤지엄"/>
+                </li>
+                <li>
+                    <img src="./images/paris4.jpg" alt="에펠탑"/>
+                </li>
+                <li>
+                    <img src="./images/italy2.jpg" alt="콜로세움"/>
+                </li>
+                <li>
+                    <img src="./images/italy1.jpg" alt="타르트"/>
+                </li>
+                <li>
+                    <img src="./images/italy5.jpg" alt="부라노섬의 항구"/>
+                </li>
+            </ul>
+        </div>
+    </section>
+    <section class="hyperLink">
+        <p>💎 어디로 갈까요? 🛩️💎 <br><br>
+            <a href="#london" class="link">London</a>
+            <a href="#paris" class="link">Paris</a>
+            <a href="#italy" class="link">Italy</a>
```

Comment:
```suggestion
    <ul>
            <li><a href="#london" class="link">London</a></li>
            <li><a href="#paris" class="link">Paris</a></li>
            <li><a href="#italy" class="link">Italy</a></li>
     </ul>
```
이렇게 ul - li 쓰는 것도 좋을 것 같아요!

Reply:
아하 이렇게 쓰면 코드가 더 구역으로 나뉘어진 게 보여서 더 좋네요!! 수정하겠습니다 감사합니다!👍☺️